### ISSUE TITLE
Migrations manager deletes existing content in Neo fields

### ISSUE THREAD

**Issue by si-em** [Firstborn/Craft-CMS-Migration-Manager#28]
I'm using Migrations Manager and it's great. Lately I've been having an issue with migrating content with the `Neo` field type. Fields and descriptions etc are updated, but all content in `Neo` fields disappears after migrating.

All migrations report that they're running as expected, but the file size for migrations with Neo fields involved are suspiciously large.

**Comment by fb-derrick-grigg** [comment]
@semonje are you running the latest plugin version? The migration file should be larger than normal as Neo field adds a lot of overhead relative to other fields. I retested Neo content migrations and all data exported/imported correctly. I'm happy to look at any sample files you can provide. I would need the section/field migration and then the content migration.

**Comment by si-em** [comment]
I'm Using Craft 2.6.3015 with Migrations Manager 1.0.8.9.

Just now I created a dummy Neo field with simple components to test it, and pushed this to our production environment:
![image](https://user-images.githubusercontent.com/17279882/38501049-58d9dfca-3c0c-11e8-8272-c87e7c094e3a.png)

Then I downloaded a backup of our production server and restored that one locally to ensure our environments were the same.

I then created a new module in the Neo field and created a new migration:
![image](https://user-images.githubusercontent.com/17279882/38501208-b6007268-3c0c-11e8-9cd7-307a8f5dea2f.png)

After pushing this to production, all the content I created was gone:
![image](https://user-images.githubusercontent.com/17279882/38501265-d5e0bb4c-3c0c-11e8-99c1-b9cd5b81ce08.png)

I did not edit any other related fields before the migration, just added a new block to the `modules2` Neo field.

**Comment by fb-derrick-grigg** [comment]
Ok, I will test that. So its the Neo setting migration that is wiping out content, not the actual content migration.

**Comment by si-em** [comment]
Just realised that we're using Craft localization. Can that affect migrations/delete content?

**Comment by fb-derrick-grigg** [comment]
No, localization are 100% supported

**Comment by si-em** [comment]
It seems like when you add a new block to an existing Neo field, and create a migration for that, all content in said Neo field is deleted.

**Comment by si-em** [comment]
Ok, so I just created a fresh install of Craft CMS (2.6.3015) with Neo (1.5.0) and Migration Manager (1.0.8.9). I pushed this to its own [repo here](https://github.com/semonje/neotest).

### Expectations
- I have content in a `neo` field on the production server.
- I want to add a new block to my `neo` field locally, create a migration (via _Migrations manager_, select my `neo` field, and _create migration_), and push it to my production environment.
- When I push this to my production environment, I expect the new block to be added and the original content to be left as is, unmodified (and not deleted).

### What I do
1. Create some content in my `neo` field on the production server.
2. Create a backup of that, and restore it locally (see latest backup [here](https://github.com/semonje/neotest/tree/master/craft/storage/backups)).
3. Modify my `neo` field locally, and create a migration of my `neo` field via Migrations Manager (see latest migration [here](https://github.com/semonje/neotest/tree/master/craft/plugins/migrationmanager/migrations/generated)).
4. Push the migration to my production environment.
5. Run the migration in my production environment.
6. Check if the new block has been added, and if the content is still there.

### What happens
- Changes to my `neo` field migrates as expected (i.e. the new block has been added)
- My original content that was stored in my `neo` field (step 1) has been deleted

In other words: the migrations run as expected when considering adding new blocks, modifying descriptions and block names in `neo` fields, but delete _existing content_ stored in the modified field. I expect its content to _not_ get deleted.

**Comment by fb-derrick-grigg** [comment]
@semonje try the latest version. The Neo field exports were removing and recreating the Neo blocktypes each time which caused any entries with Neo fields to lose their field entry data.

**Comment by si-em** [comment]
@fb-derrick-grigg Tried with the latest version (1.0.9.0), and now it works!

Awesome support. Thanks a lot! 🙏🙌
